package.json

```diff
@@ -13,7 +13,8 @@
   "main": "./base",
   "scripts": {
     "prebuild": "yarn run build:clean",
-    "build:clean": "rimraf build rules customRules base.js react.js untyped.js",
+    "clean": "rimraf build rules customRules '*.js'",
```

Comment:
Didn't know that was our convention. I'll keep that in mind for the future.

I'll get this changed up tomorrow.